src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,222 @@
+        OrderTable sittingOrderTable = orderTableService.sit(orderTable.getId());
+
+        // then
+        assertAll(
+            () -> assertNotNull(sittingOrderTable),
+            () -> assertEquals(orderTable.getId(), sittingOrderTable.getId()),
+            () -> assertTrue(sittingOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("사용자는 `주문 테이블`을 비울 수 있다.")
+    @Test
+    void clear() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+        orderTable.setOccupied(true);
+
+        when(orderTableRepository.findById(orderTable.getId())).thenReturn(
+            Optional.of(orderTable));
+        when(orderRepository.existsByOrderTableAndStatusNot(orderTable,
+            OrderStatus.COMPLETED)).thenReturn(false);
+
+        // when
+        OrderTable clearedOrderTable = orderTableService.clear(orderTable.getId());
+
+        // then
+        assertAll(
+            () -> assertNotNull(clearedOrderTable),
+            () -> assertEquals(orderTable.getId(), clearedOrderTable.getId()),
+            () -> assertEquals(0, clearedOrderTable.getNumberOfGuests()),
+            () -> assertFalse(clearedOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("`주문 테이블`을 비울 때 주문이 완료되지 않았으면 예외가 발생한다.")
+    @Test
+    void clearWithNotCompletedOrder() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+        orderTable.setOccupied(true);
+
+        when(orderTableRepository.findById(orderTable.getId())).thenReturn(
+            Optional.of(orderTable));
+        when(orderRepository.existsByOrderTableAndStatusNot(orderTable,
+            OrderStatus.COMPLETED)).thenReturn(true);
+
+        // when & then
+        assertThatThrownBy(() -> orderTableService.clear(orderTable.getId()))
+            .isInstanceOf(IllegalStateException.class);
+    }
+
+    @DisplayName("사용자는 `주문 테이블`의 손님 수를 변경할 수 있다.")
+    @Test
+    void changeNumberOfGuests() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+        orderTable.setOccupied(true);
+
+        final OrderTable request = new OrderTable();
+        request.setNumberOfGuests(4);
+
+        when(orderTableRepository.findById(orderTable.getId())).thenReturn(
+            Optional.of(orderTable));
+
+        // when
+        OrderTable changedOrderTable = orderTableService.changeNumberOfGuests(orderTable.getId(),
+            request);
+
+        // then
+        assertAll(
+            () -> assertNotNull(changedOrderTable),
+            () -> assertEquals(orderTable.getId(), changedOrderTable.getId()),
+            () -> assertEquals(request.getNumberOfGuests(), changedOrderTable.getNumberOfGuests())
+        );
+    }
+
+    @DisplayName("`주문 테이블`의 손님 수를 변경할 때 음식이 없으면 예외가 발생한다.")
+    @Test
+    void changeNumberOfGuestsWithEmptyOrder() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
```

Comment:
해당 테스트 케이스는 음식이 없어서 예외가 발생하는게 맞나요? 👀
orderTable이 `occupied`하지 않은 상태이기 때문에 발생한 예외로 보이는데, 

테스트 코드와 DisplayName의 내용만 봤을 때는 왜 예외가 발생하는건지 파악이 어려운 것 같아요. 😅

두 가지 팁이 있는데요.  한 가지는 아래와 같이 변수명으로 이 객체의 특징(?)을 명시해주는 것이고 
```suggestion
        final OrderTable emptyOrderTable = OrderTableFixture.createOrderTable();
```

두 번째는 아래와 같이 메서드 파라미터로 값을 주입받음으로써 특정 속성에 들어간 값이 다르다는 사실을 인지할 수 있게 만드는 방법이에요. (Named Argument 같은 기능이 있었다면 조금 더 접근이 쉬웠을 것 같은데 자바로써는 아래와 같은 방식이 있지 않을까 싶네요)

```suggestion
        boolean isOccupied = false;
        final OrderTable emptyOrderTable = OrderTableFixture.createWithOccupied(isOccupied);
```